backtotop.js

```diff
@@ -7,7 +7,7 @@ mybutton = document.getElementById("myBtn");
 window.onscroll = function() {scrollFunction()};
 
 function scrollFunction() {
-  if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+  if (document.body.scrollTop > 885 || document.documentElement.scrollTop > 885) {
```

Comment:
# I adjusted the height at which the back to top button appears to be lower and appear only when the user reached the footer on the homepage